.github/workflows/deploy-pages-site.yml

```diff
@@ -4,49 +4,27 @@ on:
     branches: [main]
 
 permissions:
-  contents: read
-  pages: write
-  id-token: write
-
-# Allow one concurrent deployment
-concurrency:
-  group: "pages"
-  cancel-in-progress: true
-
+  contents: write
 jobs:
-  # Build job
-  build:
+  build-and-deploy:
+    concurrency: ci-${{ github.ref }} # Recommended if you intend to make multiple deployments in quick succession.
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Checkout 🛎️
         uses: actions/checkout@v3
 
-      - name: Setup Pages
-        uses: actions/configure-pages@v1
-
-      - name: Install Dependencies
-        run: npm install
-
-      - name: Build react-search Package
-        run: npm run build
+      - name: Install and Build 🔧 # This example project is built using npm and outputs the result to the 'build' folder. Replace with the commands required to build your project, or remove this step entirely if your site is pre-built.
```

Comment:
Is this comment relevant? Looks like copy-paste from an example, so if it's still helpful maybe we can reword it, e.g. refer to this project instead of an example project?

Reply:
[ee6d8d5](https://github.com/vectara/react-search/pull/47/commits/ee6d8d57dd7edd6932517faff143460af2bca947)